source/template.yaml

```diff
@@ -0,0 +1,615 @@
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
I don't think this file was meant to have been added back? It was removed in #36.